### DISCUSSION TITLE
Issues with school email verification.

### DISCUSSION THREAD

**Discussion by Calosis** [community/community#54163] [GitHub Education]
### Hi everyone,

Hello,

I recently got an email asking me to re-verify my account. I had previously this year renewed my account status (Feb 13, 2023) and had no problem. Now, when going through the re-verify status (https://education.github.com/discount_requests/application), I cannot submit an application due to the email it wants me to use.

There are two problems with this:
1.) My school email does not allow any external emails outside our domain (@sacollege.org.za) so I am unable to verify the status of my school-issued email.
2.) The school email it wants me to use does not exist. It wants an email from: mail.sacshigh.org.za or sacshigh.org.za.
Our email addresses are not linked to this domain but rather @sacollege.org.za - The school in question is SACS High (South African College Schools High)

I have never had issues in the past with this, are you able to help me rectify this? I am more than happy to re-upload my school subject schedule as I had done in the past to first apply, then renew my GitHub Student perks.

Thank you.

**Comment by prjailani** [comment]
I am also facing the same issue. If you will be able to solve the issue then please let me know.

**Comment by jgonzalezalbisu** [comment]
Very similar issue here, the only difference is that my university doesn't give students or alumni an email address with their domain. I've posted another discussion here https://github.com/orgs/community/discussions/53695

**Comment by Irian** [comment]
I'm facing the same issue as @jgonzalezalbisu, which is strange as the documentation says that a prerequisite is to 'Have a verifiable school-issued email address **or upload documents that prove your current student status**'. I cannot reach the step to upload the documents as the system requires a school-issued email address, but I don't get one from my university.

**Comment by Calosis** [comment]
Is there just no support for GitHub Education anymore? I waited a month in a support ticket just to be told to post about the issue here. Now another month has passed and nothing from any Github staff. Support is a joke....
